src/webview/openGettingStartedWebview.ts

```diff
@@ -0,0 +1,35 @@
+import { ViewColumn, WebviewPanel, window } from "vscode";
+import { fireEvent } from "../binary/requests/requests";
+import { createIFrameTemplate } from "./webviewTemplates";
+import { setAlreadyOpenedGettingStarted } from "../state/gettingStartedOpenedState";
+import { TABNINE_GETTING_STARTED_FOR_VSCODE_URL } from "../globals/consts";
+import { ExtensionContext } from "../preRelease/types";
+
+let panel: WebviewPanel | undefined;
+
+// eslint-disable-next-line import/prefer-default-export
+export function openGettingStartedWebview(context: ExtensionContext): void {
```

Comment:
i think it'd be good to add a `fireEvent` call here with name "getting-started-opened" or something

Reply:
I didn't add it since the binary may not be loaded at this point.
we still get the pageview event from the browser